app/experimenter/nimbus-ui/src/lib/experiment.ts

```diff
@@ -10,30 +10,37 @@ export function getStatus(
   experiment?: getExperiment_experimentBySlug | getAllExperiments_experiments,
 ) {
   const status = experiment?.status;
+
   const ending = NimbusExperimentStatus.LIVE && !!experiment?.isEndRequested;
-  const released = status
-    ? [NimbusExperimentStatus.LIVE, NimbusExperimentStatus.COMPLETE].includes(
-        status,
-      )
-    : false;
+
+  // The experiment is in review, preview, or accepted state
+  const preparation = [
+    NimbusExperimentStatus.REVIEW,
+    NimbusExperimentStatus.ACCEPTED,
```

Comment:
Does this need a check for `NimbusExperimentStatus.PREVIEW` (comment on L16 above implies "review", "preview", or "accepted").

Reply:
Oh, no I added then removed a check and forgot to update the comment